cmd/algorand-indexer/main.go

```diff
@@ -78,13 +78,18 @@ var (
 	cpuProfile     string
 	pidFilePath    string
 	db             idb.IndexerDb
```

Comment:
Why do `db` and `dbOpts` need to be global variables?

Reply:
yeah, `globalIndexerDb()` didn't turn out to be used in the pattern I first imagined, it is now `indexerDbFromFlags()` and fits better.